src/apb/engine.py

```diff
@@ -996,16 +1065,48 @@ def cmdrun_push(**kwargs):
             exit(1)
 
         print("Successfully added APB to Ansible Service Broker")
+        return
+
+    namespace = kwargs['reg_namespace']
+    service = kwargs['reg_svc_name']
+    registry_route = kwargs['reg_route']
+
+    if registry_route:
+        registry = registry_route
+    else:
+        registry = get_registry_service_ip(namespace, service)
+        if registry is None:
+            print("Failed to find registry service IP address.")
+            raise Exception("Unable to get registry IP from namespace %s" % namespace)
+
+    tag = registry + "/" + kwargs['namespace'] + "/" + dict_spec['name']
+    print("Building image with the tag: " + tag)
+    try:
+        client = docker.DockerClient(base_url='unix://var/run/docker.sock', version='auto')
+        client.images.build(path=project, tag=tag, dockerfile=kwargs['dockerfile'])
+        openshift_config.load_kube_config()
+        token = openshift_client.configuration.api_key['authorization'].split(" ")[1]
+        client.login(username="unused", password=token, registry=registry, reauth=True)
+        client.images.push(tag)
+        print("Successfully pushed image: " + tag)
+        bootstrap(broker, kwargs.get("basic_auth_username"),
+                  kwargs.get("basic_auth_password"), kwargs["verify"])
+    except docker.errors.DockerException:
+        print("Error accessing the docker API. Is the daemon running?")
+        raise
+    except docker.errors.APIError:
+        print("Failed to login to the docker API.")
+        raise
 
     if not kwargs['no_relist']:
         relist_service_broker(kwargs)
 
 
 def cmdrun_remove(**kwargs):
     if kwargs["all"]:
-        route = "/v2/apb"
+        route = "/v2/apb/"
     elif kwargs["id"] is not None:
-        route = "/v2/apb" + kwargs["id"]
+        route = "/v2/apb/" + kwargs["id"]
```

Comment:
This is causing errors without the last slash. It was doing `<route>`/v2/apb<id>`.